src/App.js

```diff
@@ -26,9 +26,22 @@ const App = () => {
     <ConfigProvider
```

Comment:
we had 4+ instances of `<ConfigProvider />` - had lots of UI/css spread out over several top & mid-level components. refactored to the top for consistency and maintainability.